src/journalQueryTemplate.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
The file naming here should match other JSON resources and use snake case

Reply:
same naming parts as the other